.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,43 @@
+name: IntegrationTest
+# Need GitHub secret: DOCKER_HUB_USER, DOCKER_HUB_SECRETS, GHCR_TOKEN
+
+on:
+  push:
+    branches:
+      - master
+      - tekton-support
+      - jenkins-client-refactor
+      - feat/*
+      - fix/*
+      - test-* # make it be easier for contributors to test
+    tags:
+      - 'v*.*.*'
+  pull_request:
+    branches:
+      - 'master'
+      - 'tekton-support'
+      - 'jenkins-client-refactor'
+      - 'feat/*'
+      - 'fix/*'
```

Comment:
No need for those branches. I prefer to keep `master` and `tekton-support`

Reply:
> No need for those branches. I prefer to keep `master` and `tekton-support`

Currently, only tekton support branch has integration test. What about keep `tekton-support` for now?

Reply:
We will merge `tekton-support` into `master`. I prefer to keep `master` in order to avoid forget it.